app/services/foreman_remote_execution/renderer_methods.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# macros to fetch information about invoked jobs
+module ForemanRemoteExecution
+  module RendererMethods
+    extend ActiveSupport::Concern
+
+    def load_job_invocations(search: '', includes: nil, preload: nil)
+      load_resource(klass: JobInvocation, search: search, permission: 'view_job_invocations', includes: includes, preload: preload)
+    end
+
+    def load_job_invocation_by_id(id, includes: nil, preload: nil)
+      search_term = "id=#{id}"
```

Comment:
nitpick, but I'd keep this small string in and would wrap the equal sign with spaces

```
load_job_invocations(search: "id = #{id}", includes: includes, preload: preload).first.first
```